File: LOO 1°Parcial/Complex/Clase Complex/Clase Complex/Clase Complex.cpp
// Clase Complex.cpp : Este archivo contiene la función "main". La ejecución del programa comienza y termina ahí.
//
#include <iostream>
#include "Complex.h"

int main()
{
    cout << " Valores de numeros\n";
    Complex n1, n2(-3,5.49f),  n3(7.24f, -9.1f);
    cout << "Numero n1: " << n1.toString() << endl;
    cout << "Numero n2: " << n2.toString() << endl;
    cout << "Numero n3: " << n3.toString() << endl;
    n1 = n2.conjugate();
    cout << "Numero n1: " << n1.toString() << endl;
    cout << endl;
    //Modulos
    cout << " Modulos\n";
    cout << "Modulo de n2: " << n2.module() << endl;
    cout << "Modulo de n3: " << n3.module() << endl;
    cout << endl;
    //operaciones basicas
    cout << " Operaciones basicas\n";
    n1.add(n2, n3);
    cout << " Suma\n (" << n2.toString() << ") + (" << n3.toString() << ") = " << n1.toString() << endl;
    cout << endl;
    n1.subs(n2, n3);
    cout << " Resta\n (" << n2.toString() << ") - (" << n3.toString() << ") = " << n1.toString() << endl;
    cout << endl;
    n1.mult(n2, n3);
    cout << " Multiplicacion\n (" << n2.toString() << ") - (" << n3.toString() << ") = " << n1.toString() << endl;
    cout << endl;
    n1.divi(n2, n3);
    cout << " Division\n (" << n2.toString() << ") - (" << n3.toString() << ") = " << n1.toString() << endl;
}

File: LOO 1°Parcial/Complex/ClaseComplex.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package testers;
import ejemplosalumnos.Complex;
public class ClaseComplex {
    public static void main(String[] args) {
        Complex A = new Complex();
        Complex B = new Complex(-3.0f, 5.49f);
        Complex C = new Complex(7.24f, -9.1f);
        System.out.println("Numero n1: " + A.toString());
        System.out.println("Numero n2: " + B.toString());
        System.out.println("Numero n3: " + C.toString());
        A = B.conjugate();
        System.out.println("Numero n1: " + A.toString());
        System.out.println("Modulo de n2: " + B.module());
        System.out.println("Modulo de n3: " + C.module());
        //probando operaciones binarias
        A.add(B, C);
        System.out.println("SUMA: (" + B.toString() + ") + (" + C.toString() + ") = " + A.toString());
        A.subs(B, C);
        System.out.println("RESTA: (" + B.toString() + ") - (" + C.toString() + ") = " + A.toString());
        A.mult(B, C);
        System.out.println("MULTIPLICACION: (" + B.toString() + ") * (" + C.toString() + ") = " + A.toString() );
        A.divi(B, C);
        System.out.println("DIVISION: (" + B.toString() + ") / (" + C.toString() + ") = " + A.toString());
    }
}